lms/templates/main_django.html

```diff
@@ -28,14 +28,14 @@
   <div class="window-wrap" dir="${static.dir_rtl()}">
     <a class="nav-skip" href="{% block nav_skip %}#content{% endblock %}">{% trans "Skip to main content" %}</a>
     {% with course=request.course %}
-      {% include "header.html" %}
+      {% include "header.html"|microsite_template_path %}
```

Comment:
I am not sure I understood what you mean, but the renderer is actually capable of including Mako templates from within a django_template. That is what the `## mako` first line does. It helps the template engine to [recognize mako templates](https://github.com/edx/edx-platform/blob/master/common/djangoapps/edxmako/makoloader.py#L44) to load them with the right library.